indexer/src/main/java/org/uniprot/store/indexer/uniprot/inactiveentry/InactiveUniProtEntry.java

```diff
@@ -11,19 +11,22 @@ public class InactiveUniProtEntry {
     private final String id;
     private final String reason;
     private final String deletedReason;
+    private final String uniParcDeleted;
     private final List<String> mergedOrDemergedAccessions;
     private static final String DEMERGED = "demerged";
 
     public InactiveUniProtEntry(
             String accession,
             String id,
             String reason,
+            String uniParcDeleted,
             List<String> mergedOrDemergedAccessions,
```

Comment:
Why are we calling this uniParc deleted? Isn't the uniprotkb entry deleted not uniparc? I find it confusing.

Reply:
What name do you suggest?

Reply:
I changed to uniParcId